plat/common/x86/lcpu.c

```diff
@@ -60,13 +60,11 @@ __lcpuid lcpu_arch_id(void)
 
 int lcpu_arch_init(struct lcpu *this_lcpu)
 {
-#ifdef CONFIG_HAVE_SMP
 	int rc;
 
 	rc = apic_enable();
```

Comment:
we should have a comment here explaining why `apic_enable` must be used even if `CONFIG_HAVE_SMP` is not enabled